draft-specifications/CapTP Specification.md

```diff
@@ -290,99 +307,100 @@ fashion, even if the messages are viewed by a malicious actor.
 
 Third party handoffs define three roles:
 
--   **Gifter**: The session sharing their `desc:import-object` or
-    `desc:import-promise`
--   **Receiver**: The session the `desc:import-object` or `desc:import-promise`
-    is being shared with.
--   **Exporter**: The session exporting the `desc:import-object` or
-    `desc:import-promise`
+- **Gifter**: The peer sharing their [Reference][Model-Reference].
+- **Receiver**: The peer the [Reference][Model-Reference] is being shared with.
+- **Exporter**: The peer exporting the [Reference][Model-Reference].
 
-## Handoffs from the Gifter's perspective
+These three peers imply three sessions connecting them:
 
-When a message is sent over a CapTP boundary that includes a reference to an
-imported object from a different session, a handoff MUST occur. The handoff is
-initated by the gifter doing two things:
+- **Gifter-Exporter**
+- **Gifter-Receiver**
+- **Exporter-Receiver**
 
--  Depositing a "gift" to the exporter's bootstrap object.
+## Handoffs from the Gifter's perspective
+
+When a message is sent over a CapTP boundary that includes a [Reference][Model-Reference]
+imported from a different session, a handoff MUST occur. The handoff is
+initated by the Gifter doing two things:
 
-and:
-1.  Creating and sending a [`desc:handoff-give`](#desc-handoff-give)
-2.  Signing the `desc:handoff-give` and wrapping it within a `desc:sig-envolope`
-3.  Replace the reference to the remote object being gifted in the message to be
-    the signed handoff give in step 2
+- Depositing a "gift" to the exporter's bootstrap object.
+- Creating and sending a signed `desc:handoff-give` to the Recipient in place of the [Reference][Model-Reference].
```

Comment:
Maybe this should use receiver, rather than recipient

Reply:
good catch thanks